packages/sources/react/src/components/VtmnAlert/VtmnAlert.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { VtmnAlertType } from './types';
+import clsx from 'clsx';
+
+export interface VtmnAlertProps
+  extends React.ComponentPropsWithoutRef<'dialog'> {
+  /**
+   * The alert's title
+   */
+  title: string;
```

Comment:
Is the `title` mandatory? I have a use-case when I don't want to show title, only some content. 
Is it necessary for accessibility purpose?